vetomint/src/lib.rs

```diff
@@ -156,14 +155,20 @@ enum ConsensusStep {
     Precommit,
 }
 
+/// All vote information in a single round
+/// prevote/precommit_total is sum of all casted voting power
```

Comment:
prevote_total and precommit_total are the sum of casted voting power for prevote and precommit step, respectively

Reply:
apply it